### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Supporting the Platform] [MAS4.3.1][HighImpact][ All the controls on the Script lab page is visible as text in High Contrast theme.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#713]
**User Experience:**
User with partial vision disability will not be able to identify the controls properly in high contrast theme.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
High Contrast Theme

**Attachments:**
![mas4 3 1 hc black-snippet](https://user-images.githubusercontent.com/33054846/32605466-8aaf410e-c577-11e7-89fb-08105e239357.png)
![mas4 3 1 hc 2-navigation](https://user-images.githubusercontent.com/33054846/32605467-8b771800-c577-11e7-8e0f-f369c4e1b793.png)
![mas4 3 1 hc black-navigation](https://user-images.githubusercontent.com/33054846/32605470-8cdb234e-c577-11e7-8383-2e4a4456ec24.png)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Navigate to Navigation page. Verify Navigation page in high contrast theme.
3. Navigate to New snippet page and verify in high contrast theme.

**Current Behavior:**
All the controls are visible as text in all the High Contrast theme.
Issue repro in all the theme and across all the pages
High Contrast Black theme
High Contrast White theme
High Contrast #1 theme
High Contrast #2 theme

**Expected Behavior:**
All the controls should be visible correctly as per the theme in the high contrast theme mode
This issue is repro throughout the application.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.3.1
